examples/Makefile

```diff
@@ -283,10 +272,7 @@ mercurial: mercurial.nix
 	nix-build mercurial.nix -o mercurial -A interpreter --show-trace -j 1
 	./mercurial/bin/python -c 'import flake8'
 	nix-shell mercurial.nix -A interpreter --run "python -c 'import flake8'"
-	if [ `cat mercurial_generated.nix | grep "a209fb6" | wc -l` != "1" ]; then \
```

Comment:
The old version of this just did't work for some reason, so I had to hack in a bash script